Programmimg_Problems 7

```diff
@@ -0,0 +1,11 @@
+for (var i = 1; i <= 10; i++) {
```

Comment:
What limit did the question set for the multiplication table?

Reply:
12

Reply:
What you have currently, will it print up to 12?

Reply:
Oh! No.
I will correct it